src/index.scss

```diff
@@ -27,7 +27,16 @@ $highlight-colour: govuk-colour('grey-4');
 @import 'section-list/index';
 @import 'sidebar/index';
 @import 'status-bar/index';
+@import 'tabs/index';
 
 .hidden {
   display: none;
 }
+
+.float-right {
+  float: right
+}
+
+.float-left {
+  float: left;
+}
```

Comment:
What are these for?

Reply:
in this particular case, aligning a link to the right of the page. However we sometimes want to float other things, perhaps should be taken out of this file and called something else? Hmm, helpers.scss? It's the kind of thing I'd expect to see at the base of any framework, so in elements or frontend toolkit, but I don't believe it is

Reply:
I don't disagree, I just couldn't see a use case anywhere, or a potential use case related to tabs so I'm resisting the addition of things that we might want at some point in the future.

Reply:
it's the `invite user` link that needs to float to the right of the tabs. And it felt wrong to add a float: right without also adding a float: left :)

Reply:
SOLD!